ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-1.9-release-4.19.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.20"
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: azure-ipi-kata
+  cron: '@yearly'
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      CUSTOM_AZURE_REGION: eastus
+      LOKI_ENABLED: "false"
+      SLEEP_DURATION: 2h
+      SPOT_INSTANCES: "false"
+      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&;C00102
+      TEST_SCENARIOS: kata
+      TEST_TIMEOUT: "75"
+    post:
+    - ref: cucushift-installer-wait
+    - chain: ipi-azure-post
+    pre:
+    - chain: ipi-conf-azure
+    - ref: ipi-conf-azure-custom-region
+    - chain: azure-provision-service-principal-minimal-permission
+    - chain: ipi-install
+    - ref: ipi-azure-rbac
```

Comment:
Shouldn't we include the @wainersm's steps to generate cms for the C00102 to work?

Reply:
Next PR!
Until then, it's all hard coded to kata